tests/integration/targets/aci_l3out_bgp_peer/tasks/main.yml

```diff
@@ -376,6 +379,7 @@
       allow_self_as_count: 3
       ttl: 4
       admin_state: disabled
+      local_as_number: 101
```

Comment:
What happens when you want to do a change of config? Did you add tests for changing the local_as_number input on an existing l3out bgp peer?

Reply:
The reason why I was asking because it seems that local_as_number is child config of the bgpPeerIp config. But I see two tasks where local_as_number is defined, but they have different peer_ip which is identifier of bgpPeerIp. I could be mistaken but to me it seems like it is not updating and creating.